src/pages/shipping.js

```diff
@@ -318,7 +317,7 @@ export const boxManifest = async (boxId, userName) => {
     
     //addEventNavBarShipment("returnToPackaging");
     //document.getElementById('boxManifestTable').appendChild(result);
-    populateBoxManifestHeader(boxId,boxJSONS);
```

Comment:
Do we still need this call to populateBoxManifestHeader with two parameters now that it has been modified to have three?